pytorch_lightning/loggers/wandb.py

```diff
@@ -154,6 +161,9 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
         assert rank_zero_only.rank == 0, 'experiment tried to log from global_rank != 0'
 
         metrics = self._add_prefix(metrics)
+        if not self.WARNING_DISPLAYED and (step is not None and step + self._step_offset < self.experiment.step):
```

Comment:
I think we have a better way to handle only once displayed Warning. Could be WarningCache. Need to check.

Reply:
shall we reimplement this one?
https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/13863ccf1f96cb912406c029c50f4253f4599aee/pl_bolts/utils/warnings.py#L7